src/displayapp/fonts/generate.py

```diff
@@ -35,10 +35,11 @@ def main():
     ap = argparse.ArgumentParser(description='auto generate LVGL font files from fonts')
     ap.add_argument('config', type=str, help='config file to use')
     ap.add_argument('-f', '--font', type=str, action='append', help='Choose specific fonts to generate (default: all)', default=[])
+    ap.add_argument('--lv-font-conv', type=str, help='Path to "lv_font_conf" executable (default: lv_font_conf)', default="lv_font_conv")
```

Comment:
it doesn't really matter, except for consistency - but consider adding a short flag too, and using it in the CMake (or converting all of them to the long versions)

Reply:
I now use the long flags in cmake